consensus/rules/src/transactions/ring_ct.rs

```diff
@@ -91,7 +90,7 @@ fn simple_type_balances(rct_sig: &RctSignatures) -> Result<(), RingCTError> {
 /// <https://monero-book.cuprate.org/consensus_rules/ring_ct/bulletproofs+.html>
 fn check_output_range_proofs(
     rct_sig: &RctSignatures,
-    verifier: &mut BatchVerifier<(), dalek_ff_group::EdwardsPoint>,
+    mut verifier: impl BatchVerifier,
```

Comment:
Can/should this be a reference?
```rust
verifier: &mut impl BatchVerifier,
```

Reply:
I made the trait impl for the single threaded verifier `&mut`